### PR TITLE
update a bunch of links

### PR DESCRIPTION
**Pull request by nschloe**
This is fixing some outdated links, found with [deadlink](https://github.com/nschloe/deadlink).
```
deadlink check README.md
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -213,7 +213,7 @@ A collection of awesome browser-side [JavaScript](https://developer.mozilla.org/
 * [spine](https://github.com/spine/spine) - Lightweight MVC library for building JavaScript applications.
 * [espresso.js](https://github.com/techlayer/espresso.js) - A minimal JavaScript library for crafting user interfaces.
 * [canjs](https://github.com/canjs/canjs) - Can do JS, better, faster, easier.
-* [react](https://facebook.github.io/react/) - A library for building user interfaces. It's declarative, efficient, and extremely flexible. Works with a Virtual DOM.
+* [react](http://reactjs.org/) - A library for building user interfaces. It's declarative, efficient, and extremely flexible. Works with a Virtual DOM.
```
Comment (gokaygurcan) [inline README.md:216]: Why not https here?

File: README.md [hunk 2]
```diff
@@ -386,7 +386,7 @@ There're also some great commercial libraries, like [amchart](https://www.amchar
 * [coddoc](http://doug-martin.github.io/coddoc/) is a jsdoc parsing library. Coddoc is different in that it is easily extensible by allowing users to add tag and code parsers through the use of coddoc.addTagHandler and coddoc.addCodeHandler. coddoc also parses source code to be used in APIs.
 * [sphinx](http://www.sphinx-doc.org/) a tool that makes it easy to create intelligent and beautiful documentation
 * [Using JSDoc](http://usejsdoc.org/)
-* [Beautiful docs](http://beautifuldocs.com/) is a documentation viewer based on markdown files.
```
Comment (gokaygurcan) [inline README.md:389]: Nice one.

**Review by gokaygurcan — CHANGES_REQUESTED** [review]

**Comment by gokaygurcan** [comment]
I wouldn't call them _dead_ except for a few. But this is a nice update, thanks.

File: README.md (hunk 1 above)
Comment (nschloe) [reply README.md]: Fixed that.

**Comment by nschloe** [comment]
> I don't know about deadlink but probably it does the same thing with different name.

Probably yeah. deadlink has the option of rewriting 301s and 308s ("permantent direct").

**Review by gokaygurcan — APPROVED** [review]

**Comment by Ashe-reset** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。

**Comment by kitety** [comment]
您好，您的邮件已经收到。

**Comment by gokaygurcan** [comment]
Looks good, thanks again.

**Pull request merged**